graphql/tests/host/data.json

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
I just pushed a fix to the data scrambler. Can you try it on this file? You can call it like
`make scramble file=graphql/tests/host/data.json`